packages/aws-cdk-lib/aws-ecs/lib/log-drivers/aws-log-driver.ts

```diff
@@ -106,6 +116,10 @@ export class AwsLogDriver extends LogDriver {
     if (props.logGroup && props.logRetention) {
       throw new Error('Cannot specify both `logGroup` and `logRetentionDays`.');
     }
+
+    if (this.props.maxBufferSize != '' && this.props.mode === AwsLogDriverMode.BLOCKING) {
```

Comment:
Since we're still in the constructor, we can just use `props` here, no need to use `this`. Additionally, let's use strict equality, i.e., `!==`, when checking for an empty string. So:
`if (props.maxBufferSize !== '' && props.mode === AwsLogDriverMode.BLOCKING)`